src/sign/index.ts

```diff
@@ -107,6 +107,13 @@ export default class Client {
       fetch(address, init)
         .then((res) => {
           if (res.ok) return resolve(res.json());
+          if (res.headers.get('content-type')?.includes('application/json'))
```

Comment:
This is already assumed in line 101 above

Reply:
that's for request no? 

here we are checking response headers 
<img width="538" alt="Untitled" src="https://github.com/snapshot-labs/snapshot.js/assets/15967809/0f39d57e-b555-4789-8e56-92049c42d013">

Reply:
Correct, my bad, I thought it was checking request headers, and not response.